orbit/pkg/packaging/windows.go

```diff
@@ -197,3 +204,87 @@ func writePowershellInstallerUtilsFile(opt Options, rootPath string) error {
 
 	return nil
 }
+
+// writeManifestXML creates the manifest.xml file used when generating the 'resource.syso' metadata
+// (see writeResourceSyso). Returns the path of the newly created file.
+func writeManifestXML(opt Options, orbitPath string) (string, error) {
+	filePath := filepath.Join(orbitPath, "manifest.xml")
+	var contents bytes.Buffer
+	if err := manifestXMLTemplate.Execute(&contents, opt); err != nil {
```

Comment:
Thanks - addressed in https://github.com/fleetdm/fleet/pull/10783/commits/4c0361bb239432b6b93fa440c884bf1bcd6700f6#diff-95043ac955352f5e6fe86545b3a5cebae5caa643a9e91da888d302e6d1c7fb98R308 - I refactored the check into a stand-alone function, also changed the signature of `writeManifestXML` so that it uses the version parts array to keep things consistent (all metadata files should report the same version information).